src/StakeRegistry.sol

```diff
@@ -495,6 +508,12 @@ contract StakeRegistry is StakeRegistryStorage {
         return _totalStakeHistory[quorumNumber].length != 0;
     }
 
+    /// @notice Updates the quorum root for the given quorum 
+    /// @dev quorumRoot+1 = h(quorumRoot, {operator stake update})
+    function _updateQuorumRoot(bytes32 operatorId, uint8 quorumNumber, uint32 blockNumber, uint96 stake) internal {
```

Comment:
just hash the stake update itself imo, not the individual components

Reply:
it will always hash with `nextUpdateBlockNumber: 0` in that case

Reply:
not when they're deregistering?

Reply:
> not when they're deregistering?

also when they're deregistering tho. `nextUpdateBlockNumber` is only set to a nonzero value when we push a new one - so if the hash is over the most recent update, the next update block number will be zero.

Reply:
oh ok then. no need